docs/dev.rst

```diff
@@ -1,20 +1,522 @@
-Develop a project
+Develop a Project
 =================
 
-Set up a dev org
+A general overview on how to develop a Salesforce project with CumulusCI.
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
+
+.. note:: Run ``cci flow info dev_org`` for a full list of the ``dev_org`` flow steps.
+
+To run the ``dev_org`` flow against the project's `default org<TODO>`_:
+
+.. code-block:: console
+
+    $ cci flow run dev_org
+
+You can also use the ``--org`` option to explicitly list the org.
+
+    Example: Run the ``dev_org`` flow against the org currently defined as ``dev`` in CumulusCI.
+
+.. code-block:: console
+
+    $ cci flow run dev_org --org dev
+
+..
+
+    Open the new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+Set the Capture State
+^^^^^^^^^^^^^^^^^^^^^
+
+To make changes to capture in an org, use the ``snapshot_changes`` task to create a snapshot. A snapshot tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
+
+.. code-block:: console
+
+    $ cci task run snapshot_changes --org dev
+
+.. note:: There's no need to run ``snapshot_changes`` if you've set up a scratch org with any of these flows:
+
+    * ``dev_org``
+    * ``dev_org_namespaced``
+    * ``qa_org``
+    * ``regression_org``
+    * ``install_beta``
+    * ``install_prod``
+    
+    These flows all run ``snapshot_changes`` as their last step.
+
+The ``list_changes`` and ``retrieve_changes`` tasks detect new metadata but ignore any prior changes.
+
+To check that the snapshot was created successfully:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+
+
+List Changes
+------------
+
+To see what components have changed in a target org:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+.. note::
+    
+    This functionality relies on Salesforce's source tracking feature, so it's only available in scratch orgs.
+
+To include/exclude components from the list using the ``--include`` or ``--exclude`` option:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --include "test.*,another_regex" --exclude "something_to_exclude"
```

Comment:
Is this correct formatting?

Reply:
I'm currently reviewing this and going to rewrite this into a few separate examples.